client/my-sites/stats/stats-card-upsell/stats-card-upsell-jetpack.tsx

```diff
@@ -2,45 +2,104 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import config from '@automattic/calypso-config';
 import page from '@automattic/calypso-router';
 import { Button } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
 import React from 'react';
-import InlineSupportLink from 'calypso/components/inline-support-link';
+import { SUPPORT_URL } from 'calypso/my-sites/stats/const';
 import { useSelector } from 'calypso/state';
 import getIsSiteWPCOM from 'calypso/state/selectors/is-site-wpcom';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
 import {
 	STATS_FEATURE_DATE_CONTROL,
 	STATS_FEATURE_UTM_STATS,
 	STATS_TYPE_DEVICE_STATS,
+	STAT_TYPE_TOP_POSTS,
+	STAT_TYPE_REFERRERS,
+	STAT_TYPE_COUNTRY_VIEWS,
+	STAT_TYPE_CLICKS,
+	STAT_TYPE_TOP_AUTHORS,
+	STAT_TYPE_EMAILS_SUMMARY,
+	STAT_TYPE_SEARCH_TERMS,
+	STAT_TYPE_VIDEO_PLAYS,
 } from '../constants';
 import { trackStatsAnalyticsEvent } from '../utils';
 import StatsCardUpsellOverlay from './stats-card-upsell-overlay';
 import { Props } from './';
 
+const supportLinkWithAnchor = ( anchor: string ) => {
+	return (
+		<a
+			href={ localizeUrl( `${ SUPPORT_URL }#${ anchor }` ) }
+			target="_blank"
+			rel="noopenner noreferrer"
+		/>
+	);
+};
+
 const useUpsellCopy = ( statType: string ) => {
 	const translate = useTranslate();
 	switch ( statType ) {
 		case STATS_FEATURE_DATE_CONTROL:
 			return translate( 'Compare different time periods to analyze your site’s growth.' );
 		case STATS_FEATURE_UTM_STATS:
+			return translate( 'Generate UTM parameters and track your campaign performance data.' );
```

Comment:
Can we pass those strings from specific components? We would avoid this growing switch, and all translations would be closer to the source that needs them.

Reply:
> Can we pass those strings from specific components?

Do you mean passing from the module component that uses the upsell? I am not sure receiving the string from the respective component would benefit management unless we have already used the exact string.

> avoid this growing switch

We can use object keys for the `statType` to map the corresponding string to alter the growing switch cases.

> all translations would be closer to the source that needs them

I think consolidating the upsell copy inside the upsell component would be straightforward. Furthermore, in that case, we may also need to prepare the string for `StatsCardUpsellWPCOM` all in the source module, which would be noise.